src/Components/tests/App.test.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable prefer-destructuring */
```

Comment:
- [ ] Make sure not to disable linter rules. These are here to help us write clean code.